content/docs/guide/gamemodes/climb.md

```diff
@@ -11,16 +11,92 @@ tags:
+Because of this "Slide" sections are usually brief and focus on keeping on the board rather than gaining enough speed to fly off.
+
+Remember that in any movement mode you can always slide up gentle slopes with just 'W' while looking at the slope!
+
+### Stamina (Jump-Land Penalty) [1.6 Climb Only]
+
+Any time the player jumps (not when they are in the air for any other reason such as falling off something) a timer starts for about 1.31 seconds.
+Whenever the player lands, they will be slowed for however much time is left on the timer.
+For reference, if a player jumps and lands on flat ground about 0.66 seconds or about half the time is taken.
+This means that jumping up to something slows you down longer since you land earlier. And jumping to something beneath will slow you less or not at all.  
+**The slowdown affects both horizontal and vertical movement: running is slower but also your jump height is reduced**.  
+**A Bhop, even a perfect one, does not circumvent the verticle movemnt penalty: Bhops will always get smaller height compared to a normal jump, thus they may not always give you the best distance or be able to reach high places**.  
+For speedrunning getting around this stamnia is a top prority for optimzing runs. For completion players only certain types of platforms make stamina a cause for concern namely ones that are to small or awkward to get a running start from a standsill.
+[This doc goes into futher detail](https://kzguide.gitlab.io/techniques/stamina/).
+
+---
+## Advanced Techniques 
+
+### Climb has some techniques of its own rarely seen outside of its gamemode.
+
+**Fast Ladder Climbing**: Ladders can be climbed by "walking into them". If you are directly facing the ladder hold W, if sideways A/D. If you face the ladder diagonally you can climb the ladder both ways. If you hold W and A/D the speeds add onto each other resulting in a much faster climb.    
+
+**Ladder Jumps**: The speed you gain from fast ladder climbing is enough to fling you into the air! This technique does not have anything to do with inputting jump (which just pushes you off the ladder) and is more simillar to surfing. Gain speed with the climb and flick off near the top. Just like surfing, the way you flick off dictactes your trajectory, and whether you trade speed for height and vice versa.  
+
+**Crouch Bhops**: Bhopping starting in a crouch position. It may be required in a map when going underneath a low celling. Doing a normal Bhop and crouching in the air will raise your collision box (you tuck in your legs), so to go underneath a hazard you must start the Bhop crounched and usually stay crouched.  ![image](../../../../assets/Crouch_Bhop.png)  
+Crouching will also limit your ablity to turn in the air and thus gain speed as well. This speed disadvantage is negated if you have already reached the Bhop speedcap on KZT since you will remain at 380 regardless of how much speed you try to gain.  
```

Comment:
> Crouching will also limit your ablity

misspelt "ability"

> This speed disadvantage is negated if you have already reached the Bhop speedcap on KZT since you will remain at 380 regardless of how much speed you try to gain.  

the speed cap does not apply in the air, you can go above 380 speed in the air.